### ISSUE TITLE
owasp-modsecurity-crs update

### ISSUE THREAD

**Issue by rafalohaki** [litespeedtech/cyberpanel#1068]
The current version in cp is 3.3.2
the latest version is 3.3.4
would be great to create something like the file which contains some package urls like phpmyadmin or owasp-modsecurity-crs, then it would be easier to make updates

**Comment by usmannasir** [comment]
Have you tested latest rules?

**Comment by rafalohaki** [comment]
yes, no problem with them. i just moved all latest files to folder and it was working

**Comment by master3395** [comment]
Thanks for the report. This issue is older and may not apply to current releases.

Please confirm:
1) CyberPanel version + build (please retest on the latest public release **2.4.9**)
2) OS name and version (whatever you run, e.g. `cat /etc/os-release`)
3) Exact steps still failing on a current install
4) Relevant logs (no passwords)

Note: the latest public release is **2.4.9**. Development branches are not recommended for reproducing bugs.

If we do not hear back within 30 days, this issue may be closed as stale. You can reopen with fresh details anytime.
